src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java

```diff
@@ -274,38 +274,38 @@ public void run() {
                 bits,
                 new Object[] { "DPD", "PMP", "DGD", 'M', ItemList.Automation_ChestBuffer_HV, 'G',
                         ItemList.Field_Generator_LV, 'D', OrePrefixes.circuit.get(Materials.HV), 'P',
-                        OrePrefixes.plateQuintuple.get(Materials.Aluminium) });
+                        OrePrefixes.plateDense.get(Materials.Aluminium) });
```

Comment:
Seems like a good time to make these consistent, I would do all triple plates for these super chest recipes